pages/configuration/resin-base-images.md

```diff
@@ -1,8 +1,65 @@
 # Resin Images Wiki
 
-This wiki page contains all information about images on Resin registry.
+This wiki page contains all information about images on Resin registry. 
 
-## Base Images:
+## <a name="image-tree"></a>Resin Image Trees
+
+This sextion describes the Resin image trees (hierarchy of images). The image tree provides a good instruction to how the images fit together to serve each device type supported by Resin.
```

Comment:
These image trees provide an overview of how all the resin base images fit together for each device type supported by Resin.